portfolio/src/main/webapp/script.js

```diff
@@ -221,4 +221,34 @@ function initializePage() {
 function toggleSentimentScore() {
   showScore = !showScore;
   displayServerContent();
+}
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/**
+ * Fetches personality data and uses it to create a chart.
+ */
+function drawChart() {
+  fetch('/personality-data').then(response => response.json())
+  .then((personalityVotes) => {
+    const data = new google.visualization.DataTable();
+    data.addColumn('string', 'Personality Type');
+    data.addColumn('number', 'Votes');
+    data.addColumn({type:'string', role:'style'});
+    Object.keys(personalityVotes).forEach((type) => {
+      data.addRow([type, personalityVotes[type], 'color:#5c7fe0']);
+    });
+
+    const options = {
+      'title': 'Personality Types',
```

Comment:
I would be consistent about declaring keys with quotes or without within the same object.